deploy/helms/legion-core/templates/metrics/graphite.yaml

```diff
@@ -67,6 +67,17 @@ spec:
         - containerPort: 8125
           name: statsd
           protocol: UDP
+        livenessProbe:
+          initialDelaySeconds: 100
+          timeoutSeconds: 10
+          httpGet:
+            path: /
+            port: 80
+        livenessProbe:
+          initialDelaySeconds: 100
+          timeoutSeconds: 10
+          tcpSocket:
+            port: 2003
```

Comment:
Different ports for two probes?

Reply:
I guess both Nginx and Carbon should be checked since they both work on the same container.